cli/pkg/release/gb.go

```diff
@@ -27,28 +27,32 @@ func CreateGbPR(version, dir string, noTag bool) (gh.PullRequest, error) {
 	if (exists != gh.Branch{}) {
 		console.Warn("Branch %s already exists", branch)
 
-		cont := console.Confirm("Do you wish to continue?")
+		cont := console.Confirm("Do you wish to continue? (The remote branch will be deleted.)")
```

Comment:
> Does it close or does it pick up the new branch?
GitHub automatically closes the PR when a branch is deleted, but not the tag. I had deleting the branch in my mind because whenever I encounter a release branch that already exists, it's been because the release script failed and I need to start the whole process over. 😅 I'll try switching to the branch and see what happens.

Reply:
When switching to the existing branch, the branch already has changes, so we get:

```
Ready to create the PR on derekblank/gutenberg? [y/n]: 
To github.com:derekblank/gutenberg
 ! [rejected]        HEAD -> rnmobile/release_1.109.0 (fetch first)
error: failed to push some refs to 'github.com:derekblank/gutenberg'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
[ERROR] error pushing the PR: exit status 1
```

So we'll need to think through the flow a bit further. We can either:

1. Switch to the existing branch, reset the existing changes, and run the current flow
2. Delete both the branch and the tag on the repo and start over
3. Clone the repo with the branch already checked out, e.g.: `git clone --branch release/x.xxx.x` 

I'm not sure which is the cleanest approach, or the most likely scenario we'd run into. WDYT? I haven't done a lot of patch releases, mostly scheduled releases, so I might not be aware of some of the more nuanced use cases. Can you think of other scenarios where the branch might already exist that we'd to account for?

Reply:
hmm I haven't had a chance to try this out but did the switch command include the `-c` flag? 
`git switch {some_branch}` should fetch and checkout the HEAD of the remote branch. In that case there shouldn't be any rejected updates. The `-c` flag will create a new local branch which would cause issues when trying to push.